### DISCUSSION TITLE
mark a known bug as "Off topic"?

### DISCUSSION THREAD

**Discussion by anija** [react-native-image-picker/react-native-image-picker#1723] [General]
There's a well known bug on M1 processors:
https://github.com/react-native-image-picker/react-native-image-picker/issues/1541

I understand the point of the user that has closed the discussion, if no one with a M1 processor helps on this, no one can fix this.

But it's not a fault of the users that are using this library with M1 processor, so i can't understand why highlighting this bug should be marked as "off topic". Mark it as "known bug" and advise users that it will not be fixed (instead of leaving unanswered questions) should be the correct way to address this.
And suggesting an alternative package for users that unfortunately have a M1 processor and won't go to their company asking to throw away a bunch of mac and buy the previous mac models, would be appreciated.

**Comment by Johan-dutoit** [comment]
I locked the issue as the responses have simply been "Facing same issue", "👍 ", etc.  It adds no value.
At the same time as locking the issue, I pinned it to the issues tab so it remains visible to users facing the same issue.

There is absolutely no problem in recommending other packages (which is why your comment wasn't removed), it's a free world, people can use what they want.

That being said, with over 20 users commenting and who knows however many it has affected, if no one with an M1 wants to debug and fix it, then it'll have to stay that way until one of the maintainers actually get an M1 to debug and fix.

**Reply by Johan-dutoit** [reply]
This comment sums it up nicely

https://github.com/expo/expo/issues/11291#issuecomment-801445510

**Reply by anija** [reply]
I think that users reporting they're facing the same problems are adding some value, the value is to see how many people are blocked because of this issue, that is relevant in order to prioritize a bug. Anyway, value or not, they are surely not "off topic".
The questions were there, unanswered, a lot before my comment, so this is why i jumped to the conclusion that was my comment the trigger to close the discussion, that's all.
